crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -574,3 +571,31 @@ impl ViewEye {
         self.center += translate;
     }
 }
+
+// Logic should be similar to `impl TypedComponentFallbackProvider<LinearSpeed> for SpatialView3D`
+impl TypedComponentFallbackProvider<LinearSpeed> for ViewEye {
```

Comment:
Interesting idea making the ViewEye the fallback provider, didn't even occur to me but makes sense logically, that way you don't have to pass in the view for using the fallback path. It is a bit weird though because the duplication of those in the view is absolutely not worth it.
It would be better to pass the &dyn fallback provider alongside the property where needed.

Reply:
This is where I may have missed something.
I thought this was need to set the widget to the right value before any user editing of the value (load a scene, click on the background of the spatial 3d view, and observe the value of the widget for the linear speed; without that, the widget would show the wrong value (zero) and not call the fallback.) This is what I meant in this comment: https://github.com/rerun-io/rerun/pull/10445#issuecomment-3036483068